packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -214,9 +229,7 @@ export const createLayerVersionCustomResource = (scope: Construct, resourceNames
         Bucket: SQLLayerManifestBucket,
         Key: key,
       },
-      // Make the physical ID change each time we do a deployment, so we always check for the latest version. This means we will never have
-      // a strictly no-op deployment, but the SQL Lambda configuration won't change unless the actual layer value changes
-      physicalResourceId: PhysicalResourceId.of(`${resourceName}-${Date.now().toString()}`),
+      physicalResourceId: physicalResourceId,
```

Comment:
nit: we generally use shorthand for property names that exactly match the variables we assign:
```suggestion
      physicalResourceId,
```